packages/spec/integration/jest-preset/__tests__/preset.js

```diff
@@ -8,4 +8,9 @@ describe('jest-preset', () => {
     const calculator = require('./setup/calculator-flow').default;
     expect(calculator(2, 40)).toEqual(42);
   });
+
+  it('allows to require hops runtime code', () => {
+    const hops = require('hops');
+    expect(hops.render).not.toBeUndefined();
```

Comment:
What about `expect(hops.render).toBeDefined()`?

Reply:
Meh, you're correct. That'd be better. Changed it and am now waiting for travis again. :D